playbooks/git-venv-clone-repos.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Clone ansible-role-reqs
```

Comment:
To avoid duplication can we not use the existing role clone approach into /etc/ansible/ceph_roles?